SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -385,6 +398,47 @@ TArray<Worker_InterestOverride> USpatialSender::CreateComponentInterest(AActor*
 	return ComponentInterest;
 }
 
+SpatialGDK::RPCsOnEntityCreation USpatialSender::PackQueuedRPCsForActor(TArray<TSharedRef<FPendingRPCParams>>* RPCList, AActor* Actor)
+{
+	ensure(RPCList != nullptr);
+
+	RPCsOnEntityCreation QueuedRPCs;
+	UClass* Class = Actor->GetClass();
+	const FClassInfo& Info = ClassInfoManager->GetOrCreateClassInfoByClass(Class);
+
+	for (TSharedRef<FPendingRPCParams> RPCParams : *RPCList)
+	{
+		if (!RPCParams->TargetObject.IsValid())
+		{
+			// The target object was destroyed before we could send the RPC.
+			continue;
+		}
+
+		const FRPCInfo* RPCInfo = Info.RPCInfoMap.Find(RPCParams->Function);
+		FUnrealObjectRef TargetObjectRef(PackageMap->GetUnrealObjectRefFromNetGUID(PackageMap->GetNetGUIDFromObject(Actor)));
+		if (TargetObjectRef != FUnrealObjectRef::UNRESOLVED_OBJECT_REF)
+		{
+			TSet<TWeakObjectPtr<const UObject>> UnresolvedObjects;
+			FSpatialNetBitWriter PayloadWriter(PackageMap, UnresolvedObjects);
+
+#if !UE_BUILD_SHIPPING
```

Comment:
I think this `#if` has been removed elsewhere in master. There's now a setting we check in the SpatialGDKSettings object.